src/main/resources/portal.properties.EXAMPLE

```diff
@@ -238,10 +238,17 @@ ehcache.xml.configuration=/ehcache-mixed.xml
 ehcache.enable.statistics=true
 ehcache.persistence.path=
 
-ehcache.RepositoryCache.maxBytesHeap=100
-ehcache.RepositoryCache.maxBytesHeapUnits=MB
-ehcache.RepositoryCache.maxBytesLocalDisk=250
-ehcache.RepositoryCache.maxBytesLocalDiskUnits=MB
+# size allocation preferences for the general persistence cache
```

Comment:
for the general cache maybe we should default to 1GB heap and X GB disc since testing showed that having heap did improve performance.

Reply:
Done ... I set 1GB for heap and 4GB for disk